content/en/docs/instrumentation/php/automatic.md

```diff
@@ -61,14 +61,18 @@ function ends it.
       dependencies:
 
       {{< ot-tabs "Linux (apt)" "macOS (homebrew)">}}
-      {{< ot-tab lang="shell">}} sudo apt-get install gcc make autoconf {{< /ot-tab >}}
+
+      {{< ot-tab lang="shell">}} sudo apt-get install gcc make autoconf
+      {{< /ot-tab >}}
+
       {{< ot-tab lang="shell">}} brew install gcc make autoconf {{< /ot-tab >}}
+
       {{< /ot-tabs >}}
-      
```

Comment:
FYI: I couldn't get Prettier to play nice here when using the usual `<!-- prettier-ignore-start -->` and end directives. But thankfully, just adding blank lines between shortcodes and shortcode start/end tags, works fine.